tutormfe/plugin.py

```diff
@@ -44,6 +44,17 @@
                 },
             },
         },
+        "RECORDS_MFE_APP": {
+            "name": "records",
+            "repository": "https://github.com/openedx/frontend-app-learner-record",
+            "port": 1990,
+            "env": {
+                "production": {
+                    "SUPPORT_URL_LEARNER_RECORDS":"",
+                    "USE_LR_MFE": "",
```

Comment:
Is there a good reason why these settings do not have these values as default upstream?

Reply:
the first settings currently only has an edX-specific default, and the second one is for a feature that currently unsupported in tutor. Would it be a better idea to just leave them out? the MFE still operates if they are not present.

Reply:
> the first settings currently only has an edX-specific default

Can we change the upstream default such that it works out of the box for everyone instead of just edX? 

> Would it be a better idea to just leave them out?

Not if they are needed, no.

Reply:
FTR, it was decided in a BTR meeting that instead of adding the learner-records MFE to Tutor (and thus enabling it by default for everyone) there would be instructions on how to enable it, either here in the tutor-mfe README, or in the learner-records MFE README.